tests/bud.bats

```diff
@@ -483,6 +507,7 @@ load helpers
 }
 
 @test "bud-http-context-dir-with-Dockerfile-post" {
+  # FIXME FIXME FIXME: this is 100% identical to the -pre test above.
```

Comment:
If you click on the expando (more context) above, you'll see that this test is identical to the `-pre` one. What was the intention here? Is it OK to collapse them both into one?

Reply:
I have no idea.  @TomSweeneyRedHat @nalind Any ideas?